EventVisualisation/View/src/Initializer.cxx

```diff
@@ -56,11 +57,13 @@ void Initializer::setup()
   auto& eventManager = EventManager::getInstance();
   eventManager.setCdbPath(ocdbStorage);
```

Comment:
The storage `settings.GetValue("OCDB.default.path", "local://$ALICE_ROOT/OCDB");` seems to be Aliroot legacy.
Are you using env file for anything else that this path? If yes, then please set the default to `o2::base::NameConf::getCCDBServer()`, if not, then you can used directly `o2::base::NameConf::getCCDBServer()`.

Reply:
The env file is used currently for some o2Eve settings (fonts, colors), but I think @jmyrcha planned to refactor it out of the code at some point. I have put `o2::base::NameConf::getCCDBServer()` as the default for now.